README.md

```diff
@@ -79,6 +87,7 @@ Example of using the [Referrers API](https://github.com/opencontainers/distribut
     trust_store: <dir_to_user_trust_store>
     allow_referrers_api: <boolean_flag_for_referrers_api>
 ```
+
```

Comment:
A super nit:
Other `For example` does not have an extra line above. Shall we add them as well?

Reply:
I am just aware of this extra line. Removed this line.